app/views/users/show.html.erb

```diff
@@ -15,6 +15,9 @@
 
   <section class="posts-sec">
     <h2>Recent Posts</h2>
+    <%if can? :create, Post %>
+      <%= link_to "New Post", new_user_post_path(@user.id) %>
```

Comment:
- [ ] This button should show only if you are on your users profile. Currently I am as user number 2 and go on user number 1 profle and click new post and the app crashes. Kindly fix it
![image](https://user-images.githubusercontent.com/62705947/209175758-a921da92-03d3-46d9-a0f9-44256f177530.png)

Reply:
Thank you for your observation, I am working on a fix now